plugin.video.catchuptvandmore/resources/lib/xmltv.py

```diff
@@ -513,9 +534,22 @@ def download_xmltv_file(country_id, day_delta=0):
             pass
 
     # Check if we need to download a fresh xmltv file
+    need_to_downlod_xmltv_file = False
     if not xbmcvfs.exists(xmltv_fp):
         Script.log("xmltv file of {} for today does not exist, let's download it".format(country_id))
-        r = urlquick.get(xmltv_url)
+        need_to_downlod_xmltv_file = True
+    else:
+        # Check if we have the last version of the file
+        current_file_md5 = compute_md5(xmltv_fp)
+        remote_file_md5 = get_remote_xmltv_md5(country_id, day_delta=day_delta)
+        print(current_file_md5)
```

Comment:
no prints please..

Reply:
Arf, I missed this one during debugging! Sorry, I will remove it.

Reply:
Done